src/views/CampusSafety/views/LostAndFound/index.tsx

```diff
@@ -109,16 +111,25 @@ const LostAndFound = () => {
     console.log(`Editing report: ${reportId}`);
   };
 
+  const handleDeleteClick = (reportId: string) => {
+    setReportToDelete(reportId);
+    setDeleteModalOpen(true);
+    console.log(reportId);
```

Comment:
Do we still need this console.log?
If so then LGTM, if not then it can be removed and that also LGTM!